### PR TITLE
Packages that will be used in the OHW19

### PR DESCRIPTION
**Pull request by ocefpaf**
Follow up from #372. This PR adds the extra packages that we will be using in the OHW19. Please let me know if this is the right file we should be modifying.

### CODE REVIEW

**Comment by ocefpaf** [comment]
Ping @rabernat and @tjcrone.

File: deployments/ocean/image/binder/environment.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ dependencies:
   - owslib
   - palettable
   - pocean-core
+  - podaacpy
```
Comment (ocefpaf) [inline deployments/ocean/image/binder/environment.yml:21]: Ping @emiliom.

**Comment by tjcrone** [comment]
@ocefpaf, this looks great. I talked to Joe and he said we are fine to merge to staging during the Pangeo conference. One question, did you confirm that all of these packages are not already installed?

**Comment by tjcrone** [comment]
A quick check seemed to indicate that owslib might already be there? Might be best not to install again?

**Comment by ocefpaf** [comment]
> One question, did you confirm that all of these packages are not already installed?

I did but as you noticed I overlooked `owslib`. Just rebased against the latest commit on `staging` and removed `owslib`.

**Pull request merged**